Models/Blog.cs

```diff
@@ -1,14 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
 namespace Blog.Models
 {
-    
     public class Blog
-{
-    public string Title { get; set; }
-    public string Body { get; set; }
-
-    public Blog(string Title, string Body)
     {
-        this.Title = Title;
-        this.Body = Body;
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Body { get; set; }
```

Comment:
No need for body because the content is in the markdown file, perhaps add a description field that is not mandatory